provisioning/roles/postgis/vars/debian-7.yml

```diff
@@ -0,0 +1 @@
+postgis_package_name: "postgresql-9.1-postgis"
```

Comment:
I had to rename this to `Debian-7.yml` (note the D) for the role to work. I guess you're using a case-insensitive filesystem.

Reply:
My filesystem lacks case-awareness indeed, sorry about that!